### DISCUSSION TITLE
UDP relay mode

### DISCUSSION THREAD

**Discussion by heiher** [heiher/hev-socks5-tunnel#251] [General]
There are two UDP relay modes in HevSocks5Tunnel:

### UDP relay over UDP

This is the standard way for UDP relay of RFC 1928. Use this mode for any socks5 server that implements the UDP-ASSOCIATE method of RFC1928.

```yaml
socks5:
  udp: udp
```

### UDP relay over TCP

This is the extended way for UDP relay over TCP that implemented by [HevSocks5Server](https://github.com/heiher/hev-socks5-server).

```yaml
socks5:
  udp: tcp
```

**Comment by Paper-Dragon** [comment]
did it support?

**Comment by heiher** [comment]
> did it support?

y
